src/runtime/button-api.js

```diff
@@ -116,6 +125,13 @@ export class ButtonApi {
     }
     button.setAttribute('title', msg(TITLE_LANG_MAP, button) || '');
     button.addEventListener('click', callback);
+    if (this.deps_.config().analyticsMode == AnalyticsMode.IMPRESSIONS) {
+      this.deps_.entitlementsManager().getEntitlements().then(entitlements => {
+        this.analyticsService_.setReadyToPay(entitlements.isReadyToPay);
+        this.analyticsService_.logEvent(
```

Comment:
Please specify the future value of isFromUserAction in a comment

Reply:
Specified that isFromUserAction should be false.